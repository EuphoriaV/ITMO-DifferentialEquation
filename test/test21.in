5.83
((6.64 * 10.0) * (3.15 + x))
2.02
3.11