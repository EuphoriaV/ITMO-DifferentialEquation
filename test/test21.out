4.912E-4